### PR TITLE
Corrections: MainMenu and CodeEditor

### PR DESCRIPTION
**Pull request by MargaritaLoseva**

### CODE REVIEW

File: documentation/user-interface/code-editor.md [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
 
-You can write coded tests using JavaScript or TypeScript. For information about the API used in tests, see the [Test API](https://devexpress.github.io/testcafe/documentation/test-api/) topic in the open-source TestCafe documentation.
+You can write coded tests using JavaScript or TypeScript. Refer to the [Test API](https://devexpress.github.io/testcafe/documentation/test-api/) topic in the open-source TestCafe documentation for information about the API used in tests.
 
 The **Code Editor** provides the following code edit features: syntax highlight, code completion, parameter hints, bracket matching, go to definition, peek definition and so on.
 
-The editor also provides the ![Run test button](../../images/user-interface/run-target-icon.svg) buttons that allow you to run an entire fixture and individual tests. These buttons are shown in the gutter for lines that define fixtures and tests.
\ No newline at end of file
+The editor also provides the ![Run test button](../../images/user-interface/run-target-icon.svg) buttons that allow you to run a fixture and individual tests. These buttons are shown in the gutter for lines that define fixtures and tests.
```
Comment (dirk-pieterse) [inline documentation/user-interface/code-editor.md]: "the gutter for lines that define fixtures and tests" The gutter?

File: documentation/user-interface/main-menu.md [hunk 2]
```diff
@@ -71,6 +72,6 @@ The **Window** menu is available on MacOS and provides the following commands:
 The **Help** menu provides the following commands:
 
 * **Welcome** - Displays the **Welcome** page.
-* **Follow TestCafe Studio on Twitter** - Opens the [TestCafe Studio's page](https://twitter.com/TestCafeStudio) on the Twitter website.
-* **About** *(available on Windows)* - Provides TestCafe Studio's version and copyright.
+* **Follow TestCafe Studio on Twitter** - Opens [TestCafe Studio's page](https://twitter.com/TestCafeStudio) on Twitter.
+* **About** *(available on Windows)* - Provides TestCafe Studio's version and copyright information.
 * **Search** *(available on MacOS)* - Allows you to search for the main menu's commands.
```
Comment (dirk-pieterse) [inline documentation/user-interface/main-menu.md]: search for commands?

File: documentation/user-interface/code-editor.md (hunk 1 above)
Comment (MargaritaLoseva) [reply documentation/user-interface/code-editor.md]: Gutter is a term

File: documentation/user-interface/main-menu.md (hunk 2 above)
Comment (MargaritaLoseva) [reply documentation/user-interface/main-menu.md]: Yes, you can type a keyword, and the input will show the menu's commands (that we describe in this topic) that match to this phrase . For example, you can type "Hide" and the Search box will show you the "Hide", "Hide all" commands.

File: documentation/user-interface/code-editor.md (hunk 1 above)
Comment (dirk-pieterse) [reply documentation/user-interface/code-editor.md]: Ok

File: documentation/user-interface/main-menu.md (hunk 2 above)
Comment (dirk-pieterse) [reply documentation/user-interface/main-menu.md]: Ok

File: documentation/user-interface/main-menu.md [hunk 3]
```diff
@@ -15,24 +15,24 @@ The main menu contains the following menus:
...
+> The set of available menu commands depends on the operating system.
 
 ## TestCafe Studio
 
 The **TestCafe Studio** menu is available on MacOS and provides the following commands:
 
-* **About** - Provides TestCafe Studio's version and copyright.
-* **Services** - Allows you to work with other applications from TestCafe Studio.
+* **About** - Provides TestCafe Studio's version and copyright information.
+* **Services** - Allows you to work with other TestCafe Studio applications.
```
Comment (MargaritaLoseva) [inline documentation/user-interface/main-menu.md]: It's incorrect. Not TestCafe Studio applications, other applications installed on your computer

**Pull request merged**